git/operations.go

```diff
@@ -191,7 +205,8 @@ func changedFiles(path, subPath, ref string) ([]string, error) {
 }
 
 func execGitCmd(dir string, keyRing ssh.KeyRing, out io.Writer, args ...string) error {
-	//	println("git", strings.Join(args, " "))
+	// println("COMMAND: git", strings.Join(args, " "))
```

Comment:
Since you've touched this line: it's probably not worth keeping this -- if we decide we need to see the arguments, we should log them.

Reply:
will remove